README.md

```diff
@@ -81,7 +81,7 @@ kOS requires the following external dependencies:
 
    - \*nix - install packages (only tested on ubuntu, your mileage may very)
 
-            sudo apt install php-mbstring php-xml php-curl php-mysql php-sqlite3 php-gd
+            sudo apt install php-mbstring php-xml php-curl php-mysql php-sqlite3 php-gd php-xdebug
```

Comment:
I haven't used this tool before. Should we add it to the docker container as well?

Reply:
not until we wire up the ports at the same time. I think it's probably easier to move to sail if we want debugging in the docker container.